src/Kovan-DssSpell.sol

```diff
@@ -70,14 +39,12 @@ contract SpellAction {
     ChainlogAbstract constant CHANGELOG =
         ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
 
-    // PSM-USDC-A
-    address constant USDC               = 0xBD84be3C303f6821ab297b840a99Bd0d4c4da6b5;
-    address constant MCD_JOIN_USDC_PSM  = 0x4BA159Ad37FD80D235b4a948A8682747c74fDc0E;
-    address constant MCD_FLIP_USDC_PSM  = 0xe9eef655494F63802e9C7A7F1006547c4De3e713;
-    address constant MCD_PSM_USDC_PSM   = 0xe4dC42e438879987e287A6d9519379936d7b065A;
-    address constant LERP               = 0x489f89E54a807BE8fe531C1663FA9A39Bbdde4F4;
-    address constant PIP_USDC           = 0x4c51c2584309b7BF328F89609FDd03B3b95fC677;
-    bytes32 constant ILK_PSM_USDC_A     = "PSM-USDC-A";
+    // UNIV2USDCETH-A
+    address constant UNIV2USDCETH             = 0x44892ab8F7aFfB7e1AdA4Fb956CCE2a2f3049619;
+    address constant MCD_JOIN_UNIV2USDCETH_A  = 0x642009AA5373F7eAFE4BC02CBdBb65a3621fB70e;
+    address constant MCD_FLIP_UNIV2USDCETH_A  = 0x7cFCc9CC7045C86aA0505808218451127dED9CCe;
+    address constant PIP_UNIV2USDCETH         = 0x627969F6fe0651a703B2d0e3a5758F9fF9B7547A;
```

Comment:
For now we use this contract but we should ask Nik if he will provide an official one for kovan.